## parameter for px4_pos_controller.cpp

## 起飞高度
Takeoff_height : 1.0
## 降落速度
Land_speed : 0.3
##　上锁高度
Disarm_height : 0.1

##　地理围栏
geo_fence:
  x_min: -100.0
  x_max: 100.0
  y_min: -100.0
  y_max: 100.0
  z_min: -0.5
  z_max: 100.0
